File: labs/lab1/src/Pereche.java
/**
 * Created by Vali on 09.10.2017.
 */
public class Pereche {
    int i;
    int j;

    public Pereche(int i, int j) {
        this.i = i;
        this.j = j;
    }

    public int getI() {
        return i;
    }

    public void setI(int i) {
        this.i = i;
    }

    public int getJ() {
        return j;
    }

    public void setJ(int j) {
        this.j = j;
    }

    @Override
    public String toString() {
        return "Pereche{" +
                "i=" + i +
                ", j=" + j +
                '}';
    }
}

File: labs/lab3_c#/ConsoleApp1/ConsoleApp1/Pereche.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    class Pereche
    {
        public int i;
        public int j;

        public Pereche(int i, int j)
        {
            this.i = i;
            this.j = j;
        }

        public int J
        {
            get => j;
            set => j = value;
        }

        public int I
        {
            get => i;
            set => i = value;
        }

        public int getI()
        {
            return i;
        }

        public void setI(int i)
        {
            this.i = i;
        }

        public int getJ()
        {
            return j;
        }

        public void setJ(int j)
        {
            this.j = j;
        }

        
        public override String ToString()
        {
            return "Pereche{" +
                   "i=" + i +
                   ", j=" + j +
                   '}';
        }
    }
}